File: converter-dotnet-core-console-cs/LoggingExample.cs
using ceTe.DynamicPDF.Conversion;
using System;

namespace converter_dotnet_core_console_cs
{
    class LoggingExample
    {
        public static void LogConfiguration()
        {
            ConversionOptions.Logging.Level = LogLevel.Debug;
            ConversionOptions.Logging.LogFilePath = @"C:\temp\LogFile.txt";
            //ConversionOptions.Logging.LogAction = DelegateMethod;
        }

        public static void DelegateMethod(LogIdentifier identifier, LogLevel logLevel, string message)
        {
            Console.WriteLine(logLevel + "|" + identifier + "|" + message);
        }
    }
}

File: converter-dotnet-core-console-vb/LoggingExample.vb
Imports ceTe.DynamicPDF.Conversion
Imports System

Public Class LoggingExample

    Public Shared Sub LogConfiguration()
        ConversionOptions.Logging.Level = LogLevel.Debug
        ConversionOptions.Logging.LogFilePath = "C:\temp\LogFile.txt"
        REM ConversionOptions.Logging.LogAction = AddressOf DelegateMethod
    End Sub

    Public Shared Sub DelegateMethod(identifier As LogIdentifier, logLevel As LogLevel, message As String)
        Console.WriteLine(logLevel.ToString() & "|" & identifier.ToString() & "|" & message)
    End Sub

End Class

